frontend/src/pages/admin/TeamTypes/dialogs/TeamTypeEditDialog.vue

```diff
@@ -142,6 +142,14 @@ export default {
                         this.input.properties.trial.instanceType = '_'
                     }
                     this.input.order = '' + (teamType.order || 0)
+
+                    // Apply default feature values if undefined
+                    if (this.input.properties.features['shared-library'] === undefined) {
+                        this.input.properties.features['shared-library'] = true
+                    }
+                    if (this.input.properties.features.projectComms === undefined) {
```

Comment:
The two different syntaxes (square brackets vs dot notation) threw me slightly here. Why are the two features not using the same format? (snakeCase vs kebab-case)

Reply:
Because the underlying feature flags were originally added with this inconsistency.